docs/reference/painless-api-reference/Math.asciidoc

```diff
@@ -0,0 +1,46 @@
+////
+Automatically generated by PainlessDocGenerator. Do not edit.
+Rebuild by running `gradle generatePainlessApi`.
+////
+
+* [[painless-api-reference-Math]]Math[afterthought]##(link:{java8-javadoc}/java/lang/Math.html#Math[reference])##
```

Comment:
I decided to generate a file per class rather than dump them all in the same file because I thought these files were dense enough as is.